tasks/main.yml

```diff
@@ -22,7 +22,9 @@
     status_code: 200, 201, 208, 301, 302
   when: sitewards_server_suicide_warm_up_url != ''
 
-- cron:
+- name: Create crontask to check for server activity in specified period.
+  when: sitewards_server_suicide_active|bool
```

Comment:
giving this a second thought it might be not entirely correct, we should rather work with `state` flag or `disabled` flag, see https://docs.ansible.com/ansible/latest/modules/cron_module.html 

Reason - if this cron task is installed and then we removed it - it breaks 

State can be `present` or `absent`, while `disabled` is a boolean.